src/@types/interfaces.ts

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export interface LanguagesInput {
+  lang: string;
```

Comment:
The brackets `[` and `]` are important in the interface (see following code snippet) since there need to by multiple language keys in the object.

```
export interface LanguagesInput {
   [lang]: string;
}
export interface LanguageMessages {
  [messageID]: string;
}
```

Reply:
I tried with brackets but it shows error